### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Salamandar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Salamandar** [comment]
!testme

File: README_fr.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Si vous n’avez pas YunoHost, regardez [ici](https://yunohost.org/#/install) po
 
 enDI est un progiciel de gestion d'entreprise. C'est l'outil de pilotage de l'activité économique et sociale entièrement dédié aux CAE ainsi qu'à toute entreprise et collectif d'entrepreneur.euse.s indépendant.e.s.
 
-### Caractéristiques :
+### Fonctionnalités :
```
Comment (alexAubin) [inline README_fr.md]:
```suggestion
### Fonctionnalités
```

File: doc/DESCRIPTION_fr.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 enDI est un progiciel de gestion d'entreprise. C'est l'outil de pilotage de l'activité économique et sociale entièrement dédié aux CAE ainsi qu'à toute entreprise et collectif d'entrepreneur.euse.s indépendant.e.s.
 
-### Caractéristiques :
+### Fonctionnalités :
```
Comment (alexAubin) [inline doc/DESCRIPTION_fr.md]:
```suggestion
### Fonctionnalités
```

File: scripts/remove [hunk 3]
```diff
@@ -16,26 +16,24 @@ source /usr/share/yunohost/helpers
 #=================================================
 
 # Remove the service from the list of services known by YunoHost (added from `yunohost service add`)
-if ynh_exec_warn_less yunohost service status $app >/dev/null
+if ynh_exec_warn_less yunohost service status "$ap"p >/dev/null
```
Comment (alexAubin) [inline scripts/remove]:
```suggestion
if ynh_exec_warn_less yunohost service status "$app" >/dev/null
```

**Pull request merged**